src/Models/Category.php

```diff
@@ -104,16 +104,11 @@ public function getParentcategoriesAttribute()
             ->get();
     }
 
+    /**
+     * {@inheritdoc}
```

Comment:
I've used the inheritdoc to indicate these are overriding functions of the Searchable trait